build-tools/automation/yaml-templates/build-macos.yaml

```diff
@@ -0,0 +1,73 @@
+parameters:
+  buildPoolName: $(MacBuildPoolName)
+  buildPoolImage: $(MacBuildPoolImage)
+  dependsOn: []
+  jobName: mac_build_create_installers
+  jobDisplayName: macOS > Build
+  provisionatorChannel: latest
+  repoAlias: self
```

Comment:
Can we do 'repositoryAlias' ? We have a similar parameter on iOS. And would be nice to be able to pass the hash, I added a template checkout here: https://github.com/xamarin/yaml-templates/blob/main/sdk-unified/steps/checkout/v1.yml

Reply:
I changed the param name to `repositoryAlias` and updated all the build jobs to use the checkout template.